test/fixtures/visitors.yml

```diff
@@ -0,0 +1,4 @@
+visitor_1:
+  name:  Affirmed
+  email: sample@example.com
+  password: password
```

Comment:
Wouldn't this just be a User with a different type of role instead of a separate data model?